packages/common-stellar/src/project/models.ts

```diff
@@ -1,7 +1,8 @@
 // Copyright 2020-2023 SubQuery Pte Ltd authors & contributors
 // SPDX-License-Identifier: GPL-3.0
 
-import {forbidNonWhitelisted} from '@subql/common';
+import {forbidNonWhitelisted, ProcessorImpl} from '@subql/common';
+import {Processor, FileReference} from '@subql/types-core';
 import {
```

Comment:
For line 78, `TransactionHandler` do we need `@forbidNonWhitelisted` for filter, same as `EventHandler`